src/translateText.py

```diff
@@ -18,7 +18,7 @@ def translator_es_en(text_to_translate):
 text_to_translate = "Ha abandonado a sus médicos, señora; bajo cuyo prácticas ha perseguido el tiempo con esperanza, y no encuentra otra ventaja en el proceso sino sólo la perdiendo la esperanza por el tiempo."
```

Comment:
Put this where you're scripting so it doesn't run everytime we import the file:
```
if __name__ == "__main__":
```

Reply:
Completed as requested